filer/admin/clipboardadmin.py

```diff
@@ -11,8 +12,9 @@
 from . import views
 
 
-NO_FOLDER_ERROR = "Can't find folder to upload. Please refresh and try again"
-NO_PERMISSIONS_FOR_FOLDER = (
+NO_PERMISSIONS = _("You do not have permission to upload files.")
+NO_FOLDER_ERROR = _("Can't find folder to upload. Please refresh and try again")
```

Comment:
Seems to be no test coverage for this specific message from what I can see. Is there a test that already exists for the NO_FOLDER_ERROR error where this message cna be tested as output?

Reply:
The test is here: https://github.com/django-cms/django-filer/blob/9687d55fb59265cf090ad5e8701345567c767f40/tests/test_admin.py#L405-L406